corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -216,6 +217,17 @@ FormplayerFrontend.on("start", function (options) {
     }
 });
 
+FormplayerFrontend.on('setAppDisplayProperties', function(app) {
+    FormplayerFrontend.DisplayProperties = app.profile.properties;
+    if (Object.freeze) {
+        Object.freeze(FormplayerFrontend.DisplayProperties);
```

Comment:
this'll ensure this global state doesn't get mutated. i was hesitant to add this state, but it was a mess to pass this around through all the various functions, especially because it doesn't work with regular cloudcare since we don't know what app we're talking about until someone selects one.